server/testdriver/core/src/main/java/org/infinispan/server/test/core/ContainerInfinispanServerDriver.java

```diff
@@ -318,6 +329,16 @@ public void kill(int server) {
       Exceptions.unchecked(() -> containers.get(server).execInContainer(INFINISPAN_SERVER_HOME + "/bin/kill.sh"));
    }
 
+   // TODO should this just replace stop?
+   public void sigterm(int server) {
+      GenericContainer container = containers.get(server);
+      CountdownLatchLoggingConsumer latch = new CountdownLatchLoggingConsumer(1, SHUTDOWN_MESSAGE_REGEX);
+      container.withLogConsumer(latch);
+      Container.ExecResult result = Exceptions.unchecked(() -> container.execInContainer(INFINISPAN_SERVER_HOME + "/bin/term.sh"));
+      System.out.printf("[%d] TERM %s\n", server, result);
```

Comment:
we want this ?

Reply:
It's what we do in the pause and resume methods. Adding a log statement doesn't result in any output in the test logs for some reason.